Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Header.css';
+
+function HeaderContent() {
```

Comment:
```suggestion 
export default function HeaderContent() {}
```

functions and classes can be exported as defaults using this syntax as well, `const` can't and can only be named exported as such (no way is correct, but I thought you might like to know because in these cases it saved me some time doing so)

Reply:
and to clarify, I meant in the same line, you can always export a const as a default using 
`export default (name)` at the bottom or also you can use
`export { (name) as default } `

`let` is a different story, and there are rules that generally wont allow for it to be exported as there are specific rules about mutating exports outside of their originating namespace.